### PR TITLE
Crystal Governance document

### PR DESCRIPTION
**Pull request by beta-ziliani**
The Crystal Governance document. This document is based on the existing experience in Python, and is fairly minimal on purpose. The immediate goal is to have something on which to based the key decision-makings processes for Crystal, and it is not meant to be written in stone.

### CODE REVIEW

**Review by jhass — APPROVED** [review]

**Review by straight-shoota — APPROVED** [review]

**Comment by straight-shoota** [comment]
Just technicalities: In order to show up on the website, there should be a link to that page in `SUMMARY.md`. I think it should go in the *Home* section. And a lower-case file name fits better with the rest of the files.

**Review by bcardiff — APPROVED** [review]
I imagined this going into the crystal-website and not into the crystal-book. It's more institutional info than references of the compiler and tools.

I agree with the content 👍.

**Comment by beta-ziliani** [comment]
I don't know what is the best place. I don't think the website is the right place, I always think of it as an entry to other places (the book, the forum, etc.). In Python they have it in the [dev](https://www.python.org/dev/peps/pep-0013/) section, as the resources that are targeted to contributors.

**Comment by straight-shoota** [comment]
In the end, it really doesn't matter much. This repo is part of the website, mounted at `/reference`.

IMO crystal-website is the part that is more about public relations.

File: docs/GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- Brian Cardiff
+- Juan Wajnerman
+
+### Mandate
+
+The steering council shall work to:
+
+1. Maintain the quality and stability of the Crystal language, compiler, std-lib and any other project under [crystal-lang organization](http://github.com/crystal-lang).
+
+2. Make contributing as accessible, inclusive, and sustainable as possible.
```
Comment (wontruefree) [inline docs/GOVERNANCE.md:27]: `inclusive` as a word has become politically divisive in the past few years is there a way to choose other language for this?

Comment (wontruefree) [reply docs/GOVERNANCE.md:27]: I am not saying the thought behind the sentiment is wrong but is there maybe a way to spell it out or something.

Comment (jhass) [reply docs/GOVERNANCE.md:27]: Please make concrete constructive proposals, otherwise I don't see this as a discussion worth having.

Comment (z64) [reply docs/GOVERNANCE.md:27]: Suggestion; maybe we pick some language to defer to the standing [Code of Conduct](https://github.com/crystal-lang/crystal/blob/master/CODE_OF_CONDUCT.md) instead? I think that answers all questions as to what this bullet point stands for.

> Make contributing as accessible, inclusive, and sustainable as possible, as described by the [Code of Conduct](https://github.com/crystal-lang/crystal/blob/master/CODE_OF_CONDUCT.md)

or similar

Comment (wontruefree) [reply docs/GOVERNANCE.md:27]: Since there is a code of conduct and other language like this I am going to remove my comment.

**Comment by beta-ziliani** [comment]
@sdogruyol @waj @asterite can you please review and approve this? Many thanks.

**Review by asterite — APPROVED** [review]
This looks great, thank you for putting this up! ❤️

**Review by sdogruyol — APPROVED** [review]

**Pull request merged**